litmus-portal/frontend/src/components/InfoFilled/index.tsx

```diff
@@ -7,7 +7,7 @@ import formatCount from '../../utils/formatCount';
 import useStyles from './styles';
 
 interface CardValueData {
-  color: string;
+  color: string | undefined;
```

Comment:
Isn't color a required field! It won't really work if its undefined right?

Reply:
yes, that's correct, so as in this case we have declared a custom variable in our themes file as `totalRunsCountColor?: string;` which accepts a string | undefined value, 
so, here in this case the color accepts undefined also as we have used the above-defined label here as one of the values for color.
![Screenshot from 2020-10-06 18-39-10](https://user-images.githubusercontent.com/10995416/95205619-3dd18f00-0803-11eb-8693-bea443c9cad3.png)

Reply:
in PaletteOptions make `totalRunsCountColor: string`...remove the ? the error will go away

Reply:
You can make the `totalRunsCountColor` a required palette option, then you wouldn't need to use `undefined` in the interface.

Reply:
yes, declaring totalRunsCountColor: string in palette option worked, Thanks @S-ayanide @arkajyotiMukherjee for the feedback.